openslides_votecollector/api.py

```diff
@@ -75,28 +69,37 @@ def get_keypads():
 
 def get_device_status():
     server = get_server()
-    return server.voteCollector.getDeviceStatus()
+    try:
+        return server.voteCollector.getDeviceStatus()
+    except:
+        raise VoteCollectorError(_('No connection to VoteCollector.'))
 
 
 def start_voting(mode, options, callback_url):
     server = get_server()
     keypads = get_keypads()
 
     ext_mode = options + ';' + callback_url if options else callback_url
-    count = server.voteCollector.prepareVoting(mode + '-' + ext_mode, 0, 0, list(keypads))
-    if count < 0:
-        raise VoteCollectorError(nr=count)
+    try:
+        count = server.voteCollector.prepareVoting(mode + '-' + ext_mode, 0, 0, list(keypads))
```

Comment:
Please try to minimize the length of the try block. Try to use only one statement inside the try block. This should be the statement that may raise the exception.